### DISCUSSION TITLE
Connect AWS MSK with Strimzi

### DISCUSSION THREAD

**Discussion by amansbs** [strimzi/strimzi-kafka-operator#6562] [Q&A]
Hi everyone, 
I have a use case where my kafka cluster is running in AWS and I want strimzi to create topics in the already created Kafka cluster in AWS. Is it able to achieve that as of now and how can I do that.
Any comments/suggestions are highly appreciated.
Thanks

**Comment by scholzj** [comment]
What do you mean with _I want strimzi to create topics_? Does it mean that you want to use the Topic Operator? Or what exactly?

**Comment by amansbs** [comment]
@scholzj Yes, Thanks for the reply, What I needed was the Topic operator and it is solved now. So, I wanted to create topics in AWS MSK using strimzi and that was achievable with the Topic Operator. Also, is there any way currently by which I can create topics in multiple namespaces?
